CLDConfig/CLDReconstruction.py

```diff
@@ -55,7 +55,7 @@
 
 from Configurables import GeoSvc, TrackingCellIDEncodingSvc
 geoservice = GeoSvc("GeoSvc")
-geoservice.detectors = [os.environ["K4GEO"]+"/FCCee/CLD/compact/CLD_o2_v05/CLD_o2_v05.xml"]
+geoservice.detectors = [os.environ["K4GEO"]+"/FCCee/CLD/compact/CLD_o2_v06/CLD_o2_v06.xml"]
```

Comment:
We need to update the default simulation model as well in this case https://fcc-ee-detector-full-sim.docs.cern.ch/CLD/

Reply:
Yes we also need to update that.

Is it possible to add a default model also to `cld_steer.py`?

Reply:
apparently yes :D

I will update the documentation to also use the steering file

Reply:
But I will also update the model in the doc explicitly and then we can drop it from the example after the next release

Reply:
https://gitlab.cern.ch/fcc-ee-detector-full-sim/docs/-/merge_requests/10

Reply:
thanks!

Reply:
Documentation MR is merged